src/Components/Patient/PatientHome.tsx

```diff
@@ -91,6 +93,15 @@ export const PatientHome = (props: any) => {
     });
   };
 
+  const { data: insuranceDetials, loading: insuranceDetailsLoading } = useQuery(
+    routes.listHCXPolicies,
+    {
+      query: {
+        patient: id,
+      },
```

Comment:
Add a pagination limit of 2 to this query, we simply want the first record and also to know if there are more than 1 records right? We need not ask the BE to fetch the whole list if we are not showing it in this page.